src/Workspaces/Core/Portable/Differencing/Match.LongestCommonSubsequence.cs

```diff
@@ -17,12 +17,16 @@ internal LongestCommonSubsequence(Match<TNode> match)
                 _match = match;
             }
 
-            protected override bool ItemsEqual(IReadOnlyList<TNode> oldSequence, int oldIndex, IReadOnlyList<TNode> newSequence, int newIndex)
+            protected override bool ItemsEqual(
+                IReadOnlyList<TNode> oldSequence, int oldIndex, 
+                IReadOnlyList<TNode> newSequence, int newIndex)
             {
                 return _match.Contains(oldSequence[oldIndex], newSequence[newIndex]);
             }
 
-            internal Dictionary<TNode, TNode> GetMatchingNodes(IReadOnlyList<TNode> oldNodes, IReadOnlyList<TNode> newNodes)
+            internal Dictionary<TNode, TNode> GetMatchingNodes(
```

Comment:
Could you revert changes splitting signatures to multiple lines? I don't see how they are better. I prefer the entire signature on a single line unless it's too long, None of these are too long.